test/puppeteer/demo.test.ts

```diff
@@ -38,6 +43,26 @@ it('focuses on the menu button after pressing escape', async () => {
 	expect(await currentFocusID()).toBe('menu-button');
 });
 
+it('disables scroll by arrow key when menu is open', async () => {
+	await page.focus('#menu-button');
+	await keyboard.down('Enter');
+	await menuOpen();
+
+	await keyboard.down('ArrowDown');
+
+	expect(await page.evaluate(() => window.scrollY === 0)).toBeTruthy();
```

Comment:
I'll push up changes for this, but in general @corymharper, it's usually a better practice to not compare `window.scrollY === 0` and then use Jest to see if that's truthy, because that way doesn't give you any insight into what `window.scrollY` was if it fails the `===` test. Instead, something like the following...

```ts
expect(await page.evaluate(() => window.scrollY)).toBe(0);
```

...is better because if it fails, we know what Jest received and we can use that to determine what caused the test to fail.